pkg/config/config_test.go

```diff
@@ -8,19 +8,46 @@ import (
 
 var _ = Describe("Config", func() {
 	Context("NewConfig", func() {
-		It("Opens the file and pours it into a BlockheadConfig struct", func() {
+		It("Opens the file and pours it into a Config struct", func() {
 			config, err := NewConfig("assets/test_config.json")
 
 			Expect(err).NotTo(HaveOccurred())
 			Expect(config.Username).To(Equal("username"))
 			Expect(config.Password).To(Equal("password"))
 		})
+
+		It("fills in the service configuration", func() {
+			cfg, err := NewConfig("assets/service_config.json")
+			Expect(err).NotTo(HaveOccurred())
+			Expect(cfg.Service.Name).To(Equal("eth"))
+			Expect(cfg.Service.ID).To(Equal("some-id"))
+			Expect(cfg.Service.Description).To(Equal("some-desc"))
+			Expect(cfg.Service.Bindable).To(BeTrue())
+			Expect(cfg.Service.Tags).To(ConsistOf("eth", "geth"))
+			Expect(cfg.Service.Metadata.DisplayName).To(Equal("some-name"))
+			Expect(cfg.Service.Metadata.LongDescription).To(Equal("some-long-desc"))
+			Expect(cfg.Service.Metadata.ProviderDisplayName).To(Equal("some-provider-display-name"))
+			Expect(cfg.Service.DashboardClient.ID).To(Equal("some-client-id"))
+			Expect(cfg.Service.DashboardClient.Secret).To(Equal("some-secret"))
+			Expect(cfg.Service.Plans).To(HaveLen(1))
+			Expect(cfg.Service.Plans[0].ID).To(Equal("some-plan-id"))
+			Expect(cfg.Service.Plans[0].Name).To(Equal("free"))
```

Comment:
Change this comparing to the expected struct, instead of checking each member line by line

Reply:
sounds good.